python/lsst/summit/utils/efdUtils.py

```diff
@@ -553,3 +553,29 @@ def getSubTopics(client, topic):
     loop = asyncio.get_event_loop()
     topics = loop.run_until_complete(client.get_topics())
     return sorted([t for t in topics if t.startswith(topic)])
+
```

Comment:
Now that with this new `findTopic` function, do you still want the `getSubTopics` function above?   

If yes, do you consider changing their names?  Just by their names it's confusing what is for what.  Might be even better just to have one function taking a regular expression.

Reply:
So I do think that we still need both - here is how I imagine using them:

![image](https://github.com/lsst-sitcom/summit_utils/assets/18741538/35d749bc-f8d3-4ab2-8a54-17e7170ded14)

Reply:
And I _thought_ that the naming made the usage pretty clear, but maybe I am wrong. I don't like the idea of the regex, mainly because I wrote these to be easy, and as a self-declared regex-phobe, not only would I not be confident in how to best write that, I wouldn't even know how to use my own tools! 😄

Reply:
Given that, what do you think is best here?

Reply:
Naively I would be tempted to use a utility this way: 
```
getTopics(client, "*airflow*")  # for finding whatever containing the airflow

getTopics(client, "lsst.sal.ESS.*") # to see what lives inside ESS 
``` 

But I'm not a user of this and won't be in the short term, so I'd leave the judgement to you.

Reply:
(Actually that wasn't really regexp.. But you get my idea on wildcarding)

Reply:
Those exact examples now work 🙂

![image](https://github.com/lsst-sitcom/summit_utils/assets/18741538/92433d6f-6bed-4ea8-ad8d-dd5f41854cde)